src/test/java/edu/harvard/iq/dataverse/api/UtilIT.java

```diff
@@ -3459,6 +3459,14 @@ static Response getDatasetVersionCitation(Integer datasetId, String version, boo
         return response;
     }
 
+    static Response getFileCitation(Integer fileId, String datasetVersion, String apiToken) {
```

Comment:
Can we use the same syntax as other methods, to be more consistent and enhance readability?

We can consider `getDatasetVersionCitation` as an example.

````java
static Response getDatasetVersionCitation(Integer datasetId, String version, boolean includeDeaccessioned, String apiToken) {
        Response response = given()
                .header(API_TOKEN_HTTP_HEADER, apiToken)
                .contentType("application/json")
                .queryParam("includeDeaccessioned", includeDeaccessioned)
                .get("/api/datasets/" + datasetId + "/versions/" + version + "/citation");
        return response;
    }
````

Reply:
Well, I want to be able to pass a null API token, but sure, I'll rework it a bit.

Reply:
Done: a2194d9ce6 stop using var; make consistent with older method

(will rework if I end up adding includeDeaccessioned)